docs/usage.md

```diff
@@ -440,6 +439,8 @@ process {
 
 You will not get the final reporting outcomes of the workflow, but you will get the differential tables produced by DESeq2 or Limma, and the results of any gene seta analysis you have enabled.
 
+We have also added a dedicated pipeline parameter, `--skip_reports` that allows you to skip only the RMarkdown notebook and bundled report while leaving other reporting processes active. The `RMARKDOWNNOTEBOOK` process assumes that every grouping variable you pass to it (from the contrasts file’s variable column or PCA-derived informative_variables) exists as a valid, named column in your sample metadata. If you know your metadata or contrasts might be incomplete or non-standard (such as using formula-based yaml files), the you can use this flag to skip these steps.
```

Comment:
This seems a bit awkward.

I wonder if we should just have a global parameter for supplying grouping variables for reporting purposes, for when they can't be derived from contrasts?

Then, workflow logic is set up to only do reporting in the case of non-formula-based contrasts, or when global grouping variable are supplied. If you want a report you do one or the other.

We could still have the report-skipping for convenience (I think it might even have been asked for), but it doesn't need to be tied up with this stuff.

Reply:
> I wonder if we should just have a global parameter for supplying grouping variables for reporting purposes, for when they can't be derived from contrasts?

Makes sense! 

I also see the point that reporting is a whole separate topic to be worked on. What's your preferred way how to proceed here @pinin4fjords? Leave `--skip_reports` for now and open a ticket for reporting, or implement a parameter with grouping variable right now (but keeping it as simple as possible as reporting might change down the road anyway).

Reply:
@pinin4fjords, we need a decision pls how to move forward with this point. Is it ok to merge this with the skip flag and work on reports later separately?